tests/cases/fourslash/codeFixClassImplementInterfaceMemberTypeAlias.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts' />
+
+//// type MyType = [string, number];
+//// interface I { test(a: MyType): void; }
+//// class C implements I {[| |]}
+
+verify.rangeAfterCodeFix(`
+    test(a: [string, number]): void {
```

Comment:
Sometimes we really ought to instantiate the type reference. Consider:
``` ts
interface I<T> {
    foo(t: T): void;
}
class C implements I<number> {}
```
Then we clearly ought to use the instantiated type for `a`. 

It looks like it will be tricky to distinguish the above case and other variants from the test case.

The behavior in the test doesn't represent a regression.

Because the fix is complicated, I'll leave it for a separate PR.